docs/tutorial/part-one/index.md

```diff
@@ -1,390 +1,282 @@
 
-Try some other tricks, like the ones below:
+### 💡 Hot reloading
+Gatsby uses "hot reloading" to speed up your development process. Essentially, when you're running a Gatsby development server, the Gatsby site files are being "watched" in the background -- any time you save changes to a file, your changes will be immediately reflected in the browser, without refreshing the page!
 
-1.  Gatsby lets you add "inline styles" via a JavaScript style "prop" (we'll
-    learn about other styling options later).
+### 💡 Wait… HTML in our JavaScript?
 
-    Try replacing your page component with this:
+*If you’re familiar with React and JSX, feel free to skip this section.* If you haven’t worked with the React framework before, you may be wondering what HTML is doing in a JavaScript function. Or why we’re importing `react` but not using it anywhere. This hybrid “HTML-in-JS” is actually a syntax extension of JavaScript, for React, called JSX. You can follow along this tutorial without prior experience with React, but if you’re curious, here’s a brief primer…
 
-```jsx
-import React from "react";
+This is the JSX from the starter `/src/pages/index.js` file:
 
-export default () => <div style={{ color: `blue` }}>Hello Gatsby!</div>;
-```
 
-Change the color to "pink". Then to "tomato".
+    import React from 'react'
+    import Link from 'gatsby-link'
+    const IndexPage = () => (
+      <div>
+        <h1>Hi people</h1>
+        <p>Welcome to your new Gatsby site.</p>
+        <p>Now go build something great.</p>
+        <Link to="/page-2/">Go to page 2</Link>
+      </div>
+    )
+    export default IndexPage
 
-2.  Add some paragraph text.
+In pure JavaScript, it looks more like this:
 
-```jsx{5-6}
-import React from "react";
 
-export default () =>
- <div style={{ color: `tomato` }}>
-   <h1>Hello Gatsby!</h1>
-   <p>What a world.</p>
- </div>
-```
+    import React from 'react';
+    import Link from 'gatsby-link';
+    
+    const IndexPage = () => React.createElement(
+      'div',
+      null,
+      React.createElement(
+        'h1',
+        null,
+        'Hi people'
+      ),
+      React.createElement(
+        'p',
+        null,
+        'Welcome to your new Gatsby site.'
+      ),
+      React.createElement(
+        'p',
+        null,
+        'Now go build something great.'
+      ),
+      React.createElement(
+        Link,
+        { to: '/page-2/' },
+        'Go to page 2'
+      )
+    );
+    
+    export default IndexPage;
 
-3.  Add an image (in this case, a random one from Unsplash)
+(Now you can spot the use of the `'react'` import on several lines in the snippet above!) But wait. We’re writing JSX, right? How does the browser read that? The short answer: It doesn’t. Gatsby sites comes with tooling already set up to convert your source code into backward-compatible code that different browsers can reliably interpret. 
```

Comment:
Does this first sentence need parentheses?

Reply:
Also, @KyleAMathews this brings up the question again of how much React to teach in the tutorials.

Reply:
I think this is too long and since there are no specific actions for the learner to take, very likely to be skipped.

Reply:
When I was first learning JavaScript and React a few years ago, I was very confused for awhile that React was imported, but magically never used. This wasn't something I saw explicitly explained anywhere, and I wish it had been. Whenever I explain React to beginners, this tends to be quite a helpful side note.